### DISCUSSION TITLE
Specifying tile_size when using "extract_area"

### DISCUSSION THREAD

**Discussion by patkalie** [libvips/libvips#2461] [Q&A] [answered]
Hi,
I have used extract_area from the command line to extract areas from a .svs file as multiple .dz-files, and the tile size in the output is always 128x128. Is it possible to specify tile_size=256?

**Comment by jcupitt** [comment, marked as answer]
Hi @patkalie,

I'm not sure I understand. Do you want to change the size of the tiles that `dzsave` creates?

**Reply by patkalie** [reply]
Hi, and thanks!

No, I am able to specify tile-size when using dzsave:

```
vips dzsave D:\wsi_data\2590345.svs D:\wsi_data\0788\2590345_256 --depth one --background 243 --skip-blanks 65 --tile-size 256
```

Above dzsave command works in command-line

Below extract_area-command also works in command-line:

```
vips extract_area D:\wsi_data\2590345.svs 2590345.dz[depth=one] 0 0 10000 10000
```

but -- the tiles are 128x128

I have tried both of these without luck:

```
vips extract_area D:\wsi_data\2590345.svs 2590345.dz[depth=one, tile_size=256] 0 0 10000 10000
```

and 

```
vips extract_area D:\wsi_data\2590345.svs 2590345.dz[depth=one] 0 0 10000 10000 --tile-size 256
```

Best,
Kai

**Reply by jcupitt** [reply]
Ah, gotcha. You mustn't put spaces in the arguments inside the `[ ]` or `cmd.exe` will think it's two arguments.

Try:

```
vips extract_area D:\wsi_data\2590345.svs 2590345.dz[depth=one,tile_size=256] 0 0 10000 10000
```

You can use `tile-size` as well, which I find easier to read.

You can use spaces if you enclose the argument in quotes, but it's simpler to just not use spaces.

**Reply by patkalie** [reply]
Thanks a lot jcupitt!
Very helpful.

Best regards,
Kai
